common/db_test.go

```diff
@@ -233,13 +233,64 @@ func TestSelectUpdateOrchs_AddingUpdatingRow_NoError(t *testing.T) {
 	assert.Equal(orchs[0].ServiceURI, orch.ServiceURI)
 
 	// updating row with same orchAddress
-	orchUpdate := NewDBOrch("127.0.0.1:8937", orchAddress)
+	orchUpdate := NewDBOrch(orchAddress, "127.0.0.1:8937", 1000, 5, 10)
 	err = dbh.UpdateOrch(orchUpdate)
 	require.Nil(err)
 
 	updatedOrch, err := dbh.SelectOrchs(nil)
 	assert.Len(updatedOrch, 1)
 	assert.Equal(updatedOrch[0].ServiceURI, orchUpdate.ServiceURI)
+	assert.Equal(updatedOrch[0].ActivationRound, orchUpdate.ActivationRound)
+	assert.Equal(updatedOrch[0].DeactivationRound, orchUpdate.DeactivationRound)
+	assert.Equal(updatedOrch[0].PricePerPixel, orchUpdate.PricePerPixel)
+
+	// updating only serviceURI
+	serviceURIUpdate := NewDBOrch(orchAddress, "127.0.0.1:8938", 0, 0, 0)
```

Comment:
Let's use struct literals in this test so we can omit the optional args?

Reply:
fixed in da29fed7 & efea741f